packages/snap/src/index.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * This side-effect import puts Buffer in the global scope before
+ * any other module or snap code is loaded.
+ */
+// eslint-disable-next-line import/no-unassigned-import
+import './buffer-fix';
```

Comment:
Is it cleaner to do this in the `snap.config.js` so we don't have to have this in our source code?

e.g.

```
module.exports = {
  cliOptions: {
    src: './src/index.ts',
    port: 8080,
  },
  bundlerCustomizer: (bundler) => {
    const through = require('through2');
    bundler.transform(function () {
      let data = '';
      return through(
        function (buf, _enc, cb) {
          data += buf;
          cb();
        },
        // eslint-disable-next-line consistent-return
        function (cb) {
          this.push("globalThis.Buffer = require('buffer/').Buffer;");
          this.push(data);
          cb();
        },
      );
    });
  },
};
```